src/diffusers/__init__.py

```diff
@@ -12,6 +12,7 @@
     is_note_seq_available,
     is_onnx_available,
     is_scipy_available,
+    is_sentencepiece_available,
```

Comment:
Actually `sentencepiece` is needed by all the pipelines that need T5. Should we add them here as well?

Reply:
technically, they can use these pipelines without t5, no? i.e. use pre-computed text embeddings